diff --git a/core/lib/Drupal/Core/Asset/AssetResolver.php b/core/lib/Drupal/Core/Asset/AssetResolver.php
index c52ee7ffa..fefef06b0 100644
--- a/core/lib/Drupal/Core/Asset/AssetResolver.php
+++ b/core/lib/Drupal/Core/Asset/AssetResolver.php
@@ -264,7 +264,7 @@ public function getJsAssets(AttachedAssetsInterface $assets, $optimize) {

             // Always add a tiny value to the weight, to conserve the insertion
             // order.
-            $options['weight'] += count($javascript) / 1000;
+            $options['weight'] += count($javascript) / 3000;

             // Local and external files must keep their name as the associative
             // key so the same JavaScript file is not added twice.
